src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -152,29 +104,32 @@ export default function transformForSubmit(formConfig, form) {
       isActiveServiceDeath: transformedData?.sponsorDeathConditions,
     },
     applicants: transformApplicants(transformedData.applicants ?? []),
-    certification: {
-      date: fmtDate(new Date().toISOString().replace(/T.*/, '')) || '',
-      lastName: transformedData?.certifierName?.last || '',
-      middleInitial: transformedData?.certifierName?.middle || '',
-      firstName: transformedData?.certifierName?.first || '',
-      phoneNumber: transformedData?.certifierPhone || '',
-      relationship: transformRelationship(
-        transformedData?.certifierRelationship,
-      ),
-      streetAddress: transformedData?.certifierAddress?.streetCombined || '',
-      city: transformedData?.certifierAddress?.city || '',
-      state: transformedData?.certifierAddress?.state || '',
-      postalCode: transformedData?.certifierAddress?.postalCode || '',
-    },
+    // If certifier is also applicant, we don't need to fill out the
+    // bottom "Certification" section of the PDF:
+    certification:
+      transformedData?.certifierRelationship?.relationshipToVeteran
+        ?.applicant === true
+        ? { date: currentDate }
+        : {
+            date: currentDate,
+            lastName: transformedData?.certifierName?.last || '',
+            middleInitial: transformedData?.certifierName?.middle || '',
+            firstName: transformedData?.certifierName?.first || '',
+            phoneNumber: transformedData?.certifierPhone || '',
+            relationship: transformRelationship(
+              transformedData?.certifierRelationship,
+            ),
+            streetAddress:
+              transformedData?.certifierAddress?.streetCombined || '',
+            city: transformedData?.certifierAddress?.city || '',
+            state: transformedData?.certifierAddress?.state || '',
+            postalCode: transformedData?.certifierAddress?.postalCode || '',
+          },
```

Comment:
Updated this to either populate JUST with the current date (if certifier is also an applicant on the form). Otherwise, we fill in the `certification` object with the certifier info so the backend can stamp the lower portion of the PDF.